src/main/java/domain/game/BlackjackGame.java

```diff
@@ -0,0 +1,106 @@
+package domain.game;
+
+import domain.card.Card;
+import domain.card.GameDeck;
+import domain.card.Score;
+import domain.user.Dealer;
+import domain.user.Name;
+import domain.user.Player;
+import domain.user.User;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class BlackjackGame {
+    public static final Name dealerName = new Name("딜러");
+
+    private final List<Player> players;
+    private final Dealer dealer;
+    private final GameDeck gameDeck;
+
+    public BlackjackGame(List<Player> players, Dealer dealer, GameDeck gameDeck) {
+        this.players = new ArrayList<>(players);
+        this.dealer = dealer;
+        this.gameDeck = gameDeck;
+    }
+
+    public void drawOneMoreCardForPlayer(Player player) {
+        player.receiveCard(gameDeck.drawCard());
+    }
+
+    public void drawCardUntilDealerFinished() {
+        boolean flag = dealer.hasResult();
+        while (!flag) {
+            dealer.receiveCard(gameDeck.drawCard());
+            flag = dealer.hasResult();
+        }
+    }
+
+    public HashMap<Name, List<Card>> makeSetUpResult() {
+        HashMap<Name, List<Card>> setUpResult = new LinkedHashMap<>();
```

Comment:
```suggestion
    public Map<Name, List<Card>> makeSetUpResult() {
        Map<Name, List<Card>> setUpResult = new LinkedHashMap<>();
```
인터페이스에 대고 프로그래밍하기 라는 규칙이 있습니다 :)  
https://javacan.tistory.com/entry/OO-Basic-3-Flexibility